### PR TITLE
Replace PersonName class with Name

### PR DESCRIPTION
**Pull request by cowlinn**
Also changed CLI Syntax for start/end date to `st/et` for standardization

### CODE REVIEW

File: src/main/java/longtimenosee/model/event/Date.java [hunk 1]
```diff
@@ -18,6 +18,7 @@ public class Date {
     public static final String STANDARD_DATE = "yyyy-MM-dd";
 
     public static final String MESSAGE_FORMAT_CONSTRAINTS = "Dates must follow Format: " + STANDARD_DATE;
+    public static final String RANGE_FORMAT_CONSTRAINTS = "Ensure that Year/Month/Date values entered are appropriate.";
```
Comment (Ssatu) [inline src/main/java/longtimenosee/model/event/Date.java:21]: Maybe we could make this error message clearer.

Perhaps something like:
"Ensure that Year/Month/Date values are valid dates."

File: src/main/java/longtimenosee/model/event/Date.java [hunk 2]
```diff
@@ -34,6 +35,23 @@ public Date(String value) {
...
 
+    /**
+     * Requires a dateTime object to already have an appropriate format (YYYY-MM-DD).
+     * Checks if given Year, Month, Date values are integer values in the appropriate range
+     * (e.g: 0 <= MM <= 12)
+     * @param value in YYYY-MM-DD.
+     * @return
+     */
+    public static boolean isValidRange(String value) {
+        assert(isValidFormat(value) == true);
```
Comment (Ssatu) [inline src/main/java/longtimenosee/model/event/Date.java:46]: Good use of assert statements as a failsafe!

**Review by Ssatu — APPROVED** [review]
LGTM! Only minor wording comments so I'm going to go ahead and merge this for the purposes of advancing v1.3

File: src/main/java/longtimenosee/model/event/Date.java (hunk 2 above)
Comment (cowlinn) [reply src/main/java/longtimenosee/model/event/Date.java:46]: Actually I realized I could just do `assert(isValidFormat(value)`, bc what's inside assert is a condition!

File: src/main/java/longtimenosee/model/event/Date.java (hunk 1 above)
Comment (cowlinn) [reply src/main/java/longtimenosee/model/event/Date.java:21]: truee

**Pull request merged**